examples/powertools-examples-core/gradle/README.md

```diff
@@ -0,0 +1,34 @@
+#  Powertools for AWS Lambda (Java) - Core Utilities Example with Gradle
```

Comment:
Look @jeromevdl ! A complete working Gradle example for Java 1.8. 

It is difficult / impossible to do the "pick dependencies for the JDK given" trick that we do with maven, because the gradle wrapper itself has to change also. For the moment i've added this to the build action and run it only for Java 1.8 - see `build.yml`.